util/mbed/mbed-copy-osx.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+import os
+import time
+import shutil
+import serial
+import argparse
+import mbed_lstools as mbedls
```

Comment:
Now that we're using mbed_lstools, would it be easy to use a single script for copying on OSX and Linux?